### ISSUE TITLE
EMBTFCM Demo Alexandria 11

### ISSUE THREAD

**Issue by elcharlie** [DelphiWorlds/Kastri#94]
Hello
I have a problem with the FCM on Android with the latest realese of Alexandria, the demo FirebasCloudMessaging works for me on IOs but not on Android devices, dont receive any Token , and the EMBTFCM Demo doesnt work, for me, on IOs and Android.
I dont know what is the problem, on Sydney it works fine.
 I have verified that everything is fine in the Firebase console, I have imported the google-services.json file, I have checked the permissions, I have tried with different mobile devices,...
 Could you update the EMBTFCMDemo proyect to Alexandria?
Thanks

**Comment by DelphiWorlds** [comment]
I've now pushed a Delphi 11 version of the EMBTFCM demo

**Comment by elcharlie** [comment]
Thank you, 
I have tried on IOs device and it works perfectly, but on Android doesnt work for me, I have not received any token. the procedure PushNotificationsTokenReceivedHandler doent enter.
I dont know what is the problem.

**Comment by DelphiWorlds** [comment]
Are you importing the values from `google-services.json` correctly? The API key, Application id, Project id and GCM sender id values (in Project Options, Application > Services) should be populated, at a minimum

**Comment by elcharlie** [comment]
Yes yes, Yes, yes, correctly imported and completed. That's what seems weird to me. It is that when executing the app on android, not even the notification of the permission to receive notifications comes out. The only thing left for me is to completely reinstall the IDE
